2025-10-18 13:01:01.855 UTC [4254] LOG:  starting PostgreSQL 17.6 (Ubuntu 17.6-2.pgdg22.04+1) on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 11.4.0-1ubuntu1~22.04.2) 11.4.0, 64-bit
2025-10-18 13:01:01.856 UTC [4254] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-10-18 13:01:01.856 UTC [4254] LOG:  listening on IPv6 address "::", port 5432
2025-10-18 13:01:01.857 UTC [4254] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-10-18 13:01:01.916 UTC [4258] LOG:  database system was shut down at 2025-10-18 13:01:01 UTC
2025-10-18 13:01:01.964 UTC [4254] LOG:  database system is ready to accept connections
2025-10-18 13:01:03.139 UTC [4266] postgres@template1 LOG:  statement: 
2025-10-18 13:01:03.770 UTC [4269] postgres@template1 LOG:  statement: 
2025-10-18 13:01:04.345 UTC [4272] postgres@template1 LOG:  statement: 
2025-10-18 13:06:02.014 UTC [4256] LOG:  checkpoint starting: time
2025-10-18 13:06:02.028 UTC [4256] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 1 recycled; write=0.006 s, sync=0.001 s, total=0.015 s; sync files=2, longest=0.001 s, average=0.001 s; distance=0 kB, estimate=0 kB; lsn=0/6000130, redo lsn=0/60000D8
2025-10-18 13:10:28.180 UTC [4256] LOG:  checkpoint starting: force wait
2025-10-18 13:10:28.186 UTC [4256] LOG:  checkpoint complete: wrote 0 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.001 s, sync=0.001 s, total=0.006 s; sync files=0, longest=0.000 s, average=0.000 s; distance=16383 kB, estimate=16383 kB; lsn=0/7000080, redo lsn=0/7000028
2025-10-18 13:15:28.286 UTC [4256] LOG:  checkpoint starting: time
2025-10-18 13:15:28.300 UTC [4256] LOG:  checkpoint complete: wrote 0 buffers (0.0%); 0 WAL file(s) added, 0 removed, 2 recycled; write=0.001 s, sync=0.001 s, total=0.015 s; sync files=0, longest=0.000 s, average=0.000 s; distance=16384 kB, estimate=16384 kB; lsn=0/80000B8, redo lsn=0/8000060
2025-10-18 13:18:33.438 UTC [4435] postgres@postgres LOG:  statement: select pid, client_addr, state, sync_state from pg_stat_replication;
2025-10-18 13:41:47.072 UTC [4598] postgres@postgres ERROR:  syntax error at or near "frompg_stat_replication" at character 86
2025-10-18 13:41:47.072 UTC [4598] postgres@postgres STATEMENT:  select application_name, pg_wal_lsn_dff(pg_current_wal_lsn(),replay_lsn) as byte_lag frompg_stat_replication;
2025-10-18 13:42:09.325 UTC [4598] postgres@postgres LOG:  statement: select application_name, pg_wal_lsn_dff(pg_current_wal_lsn(),replay_lsn) as byte_lag from  pg_stat_replication;
2025-10-18 13:42:09.329 UTC [4598] postgres@postgres ERROR:  function pg_wal_lsn_dff(pg_lsn, pg_lsn) does not exist at character 26
2025-10-18 13:42:09.329 UTC [4598] postgres@postgres HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
2025-10-18 13:42:09.329 UTC [4598] postgres@postgres STATEMENT:  select application_name, pg_wal_lsn_dff(pg_current_wal_lsn(),replay_lsn) as byte_lag from  pg_stat_replication;
2025-10-18 13:42:55.683 UTC [4598] postgres@postgres LOG:  statement: select application_name, pg_wal_lsn_diff(pg_current_wal_lsn(),replay_lsn) as byte_lag from  pg_stat_replication;
2025-10-18 13:46:56.425 UTC [4598] postgres@postgres LOG:  statement: insert into applications (full_name, gender, phone, email, date, bio, agreement, user_id) values ('rep_test', 'M', '+7000', 't@t','2025-10-12', 'b', true, 1) returning id;
2025-10-18 13:46:56.428 UTC [4598] postgres@postgres ERROR:  relation "applications" does not exist at character 13
2025-10-18 13:46:56.428 UTC [4598] postgres@postgres STATEMENT:  insert into applications (full_name, gender, phone, email, date, bio, agreement, user_id) values ('rep_test', 'M', '+7000', 't@t','2025-10-12', 'b', true, 1) returning id;
2025-10-18 13:47:24.386 UTC [4628] postgres@u68824 LOG:  statement: insert into applications (full_name, gender, phone, email, date, bio, agreement, user_id) values ('rep_test', 'M', '+7000', 't@t','2025-10-12', 'b', true, 1) returning id;
2025-10-18 13:47:24.409 UTC [4628] postgres@u68824 ERROR:  insert or update on table "applications" violates foreign key constraint "applications_user_id_fkey"
2025-10-18 13:47:24.409 UTC [4628] postgres@u68824 DETAIL:  Key (user_id)=(1) is not present in table "users".
2025-10-18 13:47:24.409 UTC [4628] postgres@u68824 STATEMENT:  insert into applications (full_name, gender, phone, email, date, bio, agreement, user_id) values ('rep_test', 'M', '+7000', 't@t','2025-10-12', 'b', true, 1) returning id;
2025-10-18 13:50:28.998 UTC [4256] LOG:  checkpoint starting: time
2025-10-18 13:50:29.820 UTC [4256] LOG:  checkpoint complete: wrote 9 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.806 s, sync=0.001 s, total=0.823 s; sync files=8, longest=0.001 s, average=0.001 s; distance=0 kB, estimate=14745 kB; lsn=0/8000388, redo lsn=0/8000330
2025-10-18 13:51:09.957 UTC [4628] postgres@u68824 LOG:  statement: insert into users (login, hashed_password) values ('test_user', 'hash123') returning user_id;
2025-10-18 13:51:24.741 UTC [4628] postgres@u68824 LOG:  statement: insert into applications (full_name, gender, phone, email, date, bio, agreement, user_id) values ('rep_test', 'M', '+7000', 't@t','2025-10-12', 'b', true, 1) returning id;
2025-10-18 13:51:38.811 UTC [4628] postgres@u68824 LOG:  statement: select * from applications;
2025-10-18 13:55:28.918 UTC [4256] LOG:  checkpoint starting: time
2025-10-18 13:55:29.733 UTC [4256] LOG:  checkpoint complete: wrote 9 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.805 s, sync=0.001 s, total=0.816 s; sync files=8, longest=0.001 s, average=0.001 s; distance=1 kB, estimate=13271 kB; lsn=0/80008A8, redo lsn=0/8000850
2025-10-18 14:51:43.262 UTC [4256] LOG:  checkpoint starting: force wait
2025-10-18 14:51:43.274 UTC [4256] LOG:  checkpoint complete: wrote 0 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.002 s, sync=0.001 s, total=0.013 s; sync files=0, longest=0.000 s, average=0.000 s; distance=16381 kB, estimate=16381 kB; lsn=0/9000080, redo lsn=0/9000028
2025-10-18 14:54:10.465 UTC [4981] postgres@postgres LOG:  statement: insert into users (login, hashed_password) values ('admin', 'sdfdsfpp243j24j23knn5454') returning user_id;
2025-10-18 14:54:10.468 UTC [4981] postgres@postgres ERROR:  relation "users" does not exist at character 13
2025-10-18 14:54:10.468 UTC [4981] postgres@postgres STATEMENT:  insert into users (login, hashed_password) values ('admin', 'sdfdsfpp243j24j23knn5454') returning user_id;
2025-10-18 14:55:58.646 UTC [4984] postgres@u68824 LOG:  statement: select * from applications;
2025-10-18 14:56:43.378 UTC [4256] LOG:  checkpoint starting: time
2025-10-18 14:56:43.988 UTC [4256] LOG:  checkpoint complete: wrote 7 buffers (0.0%); 0 WAL file(s) added, 0 removed, 2 recycled; write=0.605 s, sync=0.001 s, total=0.611 s; sync files=7, longest=0.001 s, average=0.001 s; distance=16385 kB, estimate=16385 kB; lsn=0/A0005A0, redo lsn=0/A000548
